boost-maven/boost-boosters/booster-cdi20/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>boost.boosters</groupId>
+    <artifactId>cdi</artifactId>
+    <version>2.0.M1-SNAPSHOT</version>
```

Comment:
Let's add an extra digit for boost-specific changes.  

So it'd be:  
`<spec-Major>.<spec-Minor>.<rev-for-this-specific-booster>.<milestone>-?SNAPSHOT?`